Issue 2269

anisa-hawes opened the pull request:
We would like to acknowledge support from the Portuguese Foundation for Science and Technology by updating the following pages:

- [x] en/supporters.md 

- [x] pt/apoiadores.md

- [x] es/colaboradores.md

- [x] fr/nos-soutiens.md

Closes #2269 

Hello @programminghistorian/spanish-team Could you help to translate this phrase into Spanish? Would it make sense to write: Fundação para a Ciência e Tecnologia, **a través de su programa especializado** '[Verão com Ciência 2021](https://www.fct.pt/apoios/veraocomciencia/index.phtml.pt)', Portugal [septiembre 2021] ?

Hello @programminghistorian/french-team Could you help to translate this phrase into French? Would it make sense to write: Fundação para a Ciência e Tecnologia, **de l'initiative** '[Verão com Ciência 2021](https://www.fct.pt/apoios/veraocomciencia/index.phtml.pt)', Portugal [septembre 2021]


### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
  - [x] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing editor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [x] Add the appropriate "Label"
- [x] [Ensure the status checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [x] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing build errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Build Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-build-errors). Then contact the technical team if you need further help.*

anisa-hawes committed:
Update en/supporters.md

I am updating en/supporters.md to acknowledge support from the Portuguese Foundation for Science and Technology.

anisa-hawes committed:
Update pt/apoiadores.md

I am updating pt/apoiadores.md to acknowledge support from the Portuguese Foundation for Science and Technology.

rivaquiroga commented:
@anisa-hawes, the Spanish translation is correct

anisa-hawes committed:
Update es/colaboradores.md

I am updating es/colaboradores.md to acknowledge support from the Portuguese Foundation for Science and Technology.

spapastamkou commented:
Hi Anisa, you can replace "de l'initiative" with either "à travers le projet" OR "à travers le programme de soutien".  Hope this helps.

anisa-hawes committed:
Update fr/nos-soutiens.md

I am updating fr/nos-soutiens.md to acknowledge support from the Portuguese Foundation for Science and Technology.

anisa-hawes committed:
Update detecting-text-reuse-with-passim.md

I am updating the link at line 69 of `en/lessons/detecting-text-reuse-with-passim.md`, replacing https://www.oracle.com/ch-de/java/technologies/javase/javase-jdk8-downloads.html (broken) with the live link https://www.oracle.com/java/technologies/downloads/#java8
